src/Microsoft.AspNetCore.Sockets.Http/Internal/SocketLoggerExtensions.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.AspNetCore.Sockets.Internal
 {
-    public static class SocketLoggerExtensions
+    internal static class SocketLoggerExtensions
```

Comment:
Was to avoid collisions with the other SocketLoggerExtensions classes in the other packages